pulpcore/cli/ansible/distribution.py

```diff
@@ -36,7 +36,8 @@ def _repository_callback(
 ) -> EntityFieldDefinition:
     # Pass None and "" verbatim
     if value:
-        pulp_ctx: PulpContext = ctx.find_object(PulpContext)
+        pulp_ctx = ctx.find_object(PulpContext)
+        assert pulp_ctx is not None
```

Comment:
These kind of changes are actually a good sign, because they mean that click declared a proper return type `Optional[PulpContext]` instead of `Any`.

Reply:
What prompted this change?

Reply:
Click8 changed the return-type of `Context.find_object` from `Any` to `Optional[WhateverTypeYouRequested]`.
So click is more accurate, and we need to make sure it's not returning `None`. Or mypy will complain.